tensorflow/python/compiler/tensorrt/trt_convert.py

```diff
@@ -150,6 +151,10 @@ class TrtConversionParams(
       number of cached engines is already at max but none of them supports the
       input shapes, the TRTEngineOp will fall back to run the original TF
       subgraph that corresponds to the TRTEngineOp.
+    enable_sparse_compute: only effective with TensorRT 8 and newer with an
+      NVIDIA Ampere GPU or newer. It allows TensorRT to take advantage of sparse
+      compute capability of TensorCores. By default, this parameter is set to
+      True. Can be turned off for debuging reason.
```

Comment:
If turning off the feature is only for debugging, shall we avoid an API change for this?

Reply:
I'm gonna fix my comment

Reply:
Fixed